developerportal/apps/common/tests/test_utils.py

```diff
@@ -1,11 +1,23 @@
 import datetime
```

Comment:
A lot of the code in here is just moved from the redundant and just-deleted `test_common.py`. I'll mark the bits which are new